server/sonar-server/src/test/resources/org/sonar/server/source/ws/ScmActionTest/accept_negative_value_in_from_parameter.json

```diff
@@ -0,0 +1,15 @@
+{
+  "scm": [
+    [1, "julien", "2015-03-30T12:34:56+0000", "123-456-789"],
+    [2, "julien", "2015-03-29T12:34:56+0000", "123-456-710"],
+    [3, "julien", "2015-03-27T12:34:56+0000", "456-789-101"]
+  ]
+}
+
```

Comment:
It might be a display artifact, but why are there so many empty lines in your JSON fixtures?